include/antler/system/utils.hpp

```diff
@@ -47,23 +55,15 @@ namespace antler::system {
       return s;
    }
 
-   template <typename Enum, typename = std::enable_if_t<std::is_enum_v<Enum>>>
-   inline static std::string_view to_string(Enum e) { return magic_enum::enum_name(e); }
-
-   template <typename Enum, typename = std::enable_if_t<std::is_enum_v<Enum>>>
-   inline static Enum from_string(std::string_view s) { 
-      auto sv = magic_enum::enum_cast<Enum>(s);
-      if (sv) {
-         return *sv;
-      } else {
-         return Enum::error;
+   inline static int32_t execute(std::string cmd, const std::vector<std::string>& args) {
+      std::string_view prog = cmd;
+      for (const auto& arg : args) {
+         cmd += " " + arg;
       }

```

Comment:
This needs to be fixed: prog may not be valid if cmd is resized. 
```suggestion
   inline static int32_t execute(std::string_view prog, const std::vector<std::string>& args) {
      std::string cmd{prog};
      for (const auto& arg : args) {
         cmd += " " + arg;
      }
```